github_visualization/github_stats.py

```diff
@@ -0,0 +1,289 @@
+import requests
+
+
+class GithubStatFetcher:
+    """A GitHub statistics fetcher for github repositories
+    """
+
+    def __init__(self, user_name, repositoty_name):
+        ''' Initialize GithubStatFetcher Object
+
+        Parameters
+        -------------
+        user_name : str
+           Name of the user or organization of the repository owner
+        repositoty_name : str
+           Name of repository
+        '''
+        self.user_name = user_name
+        self.repositoty_name = repositoty_name
+
+        self.baseurl = "https://api.github.com/repos/%s/%s" % (
+            self.user_name, self.repositoty_name)
+
+    def get_total_contributions(self, weeks):
+        ''' Helper function to calculate total additions
+        and total deletions from a list of weeks of contributions
+        of a contributor
+
+        Parameters
+        -------------
+        weeks : list
+           List of weeks of contributions. Example:
+           [
+            {'w': 1254009600, 'a': 5, 'c': 2, 'd': 9},
+           ]
+
+        '''
+        total_additions = 0
+        total_deletions = 0
+        for week in weeks:
+            total_additions += week['a']
+            total_deletions += week['d']
+        return total_additions, total_deletions
+
+    def get_cumulative_contributors(self, contributors_list):
+        ''' Helper function to calculate total contributors as
+        new contributors join with time
+
+        Parameters
+        -------------
+        contributors_list : list
+           List of contributors with weeks of contributions. Example:
+           [
+               {
+                    'weeks': [
+                            {'w': 1254009600, 'a': 5, 'c': 2, 'd': 9},
+                        ],
+                    .....
+                },
+            ]
+
+        '''
+        contributorsJoinDate = {}
```

Comment:
This doesn't follow the style of most of your code (contributors_join_date would be more uniform). Any reason?

I see that this is the case for multiple variables underneath, such as joinDate, etc.